src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/PropertyRef.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    internal struct PropertyRef
```

Comment:
Should it be `readonly struct`?

Reply:
Yes. Thanks. The next commit will include readonly.